src/screens/LocalCommunity/index.js

```diff
@@ -186,15 +77,13 @@ const LocalCommunity = () => {
           {StringConstant.onboardingLocalCommunityHeadline}
         </Text>
         <Text style={styles.textDesc}>{StringConstant.onboardingLocalCommunitySubHeadline}</Text>
-        <FlatList
-          data={location}
-          renderItem={renderItem}
-          keyExtractor={(item, index) => index.toString()}
-        />
+        <FlatList data={location} renderItem={renderItem} keyExtractor={handleKeyExtractor} />
 
         {/* First Location */}
         {location.length <= 0 ? (
-          <TouchableNativeFeedback onPress={() => onPressFirstLocation(true)}>
+          <TouchableNativeFeedback
+            testID="onPressFirstLocation"
+            onPress={() => onPressFirstLocation(true)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The keyExtractor prop for FlatList has been changed from an inline function `(item, index) => index.toString()` to `handleKeyExtractor`. Ensure that `handleKeyExtractor` provides unique keys for each item in the list.
```diff
- keyExtractor={(item, index) => index.toString()}
+ keyExtractor={handleKeyExtractor}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->